.circleci/config.yml

```diff
@@ -43,7 +44,7 @@ commands:
       - checkout
       - run:
           name: Generate cache key
-          command: find . -type f -name "*.gradle*" -exec shasum {} + | sort > /tmp/checksum.txt && cat /tmp/checksum.txt
+          command: find . -type f -name "*.gradle*" -o -name "gradle-wrapper*" | xargs shasum | sort > /tmp/checksum.txt && cat /tmp/checksum.txt
```

Comment:
This looks quite sophisticated, I am trying to read through the code what is this doing but not much success. Do you mind adding a comment saying: `# find x | shasum y | sort by x`

Reply:
```suggestion
          # Find all gradle related files, hash each and order them consistently, then save to file for later use
          command: find . -type f -name "*.gradle*" -o -name "gradle-wrapper*" | xargs shasum | sort > /tmp/checksum.txt && cat /tmp/checksum.txt
```